ui/components/app/currency-input/index.scss

```diff
@@ -17,4 +17,41 @@
     background: none;
     border: none;
   }
+
+  &__pulsing-bar {
+    height: 4px;
+    background: #d9d9d9;
```

Comment:
Is there a [design system token](https://github.com/MetaMask/design-tokens/blob/main/src/figma/tokens.json#L329-L554) we can use for this? If not, can we add a TODO item to add it one in the future? Applies to other hardcoded hex values as well

Reply:
Just asked the designers about this

Reply:
Updated to use `icon-muted`